server/routes/index.js

```diff
@@ -1,20 +1,21 @@
-const express = require('express')
-const { formatRawProductData, formatRawThemeData } = require('../utils')
+/* eslint-disable no-console */
+const express = require('express');
+const { formatRawProductData, formatRawThemeData } = require('../utils');
 
-module.exports = function index ({ productService }) {
-  const router = express.Router()
+module.exports = function index({ productService }) {
+  const router = express.Router();
   router.get('/', async (req, res) => {
-    const rawProducts = await productService.getEntries('product')
-    const rawThemes = await productService.getEntries('themes')
-    const products = formatRawProductData(rawProducts)
-    const statuses = ['backlog', 'discovery', 'alpha', 'beta', 'live']
-    const themes = formatRawThemeData(rawThemes)
-    console.log('GET index')
+    const rawProducts = await productService.getEntries('product');
+    const rawThemes = await productService.getEntries('themes');
```

Comment:
Maybe not for now, but I think these could be fetched in parallel with 'Promise.all'.

Reply:
I haven't touched this code.